open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerAdminServices.java

```diff
@@ -733,25 +731,22 @@ public VoidResponse deleteEventBus(String userId, String serverName)
              */
             OMAGServerConfig serverConfig = configStore.getServerConfig(userId, serverName, methodName);
 
-            if (configStore != null)
-            {
-                List<String>  configAuditTrail          = serverConfig.getAuditTrail();
+            List<String>  configAuditTrail          = serverConfig.getAuditTrail();
 
-                if (configAuditTrail == null)
-                {
-                    configAuditTrail = new ArrayList<>();
-                }
+            if (configAuditTrail == null)
+            {
+                configAuditTrail = new ArrayList<>();
+            }
 
-                configAuditTrail.add(new Date().toString() + " " + userId + " deleted configuration for default event bus.");
+            configAuditTrail.add(new Date().toString() + " " + userId + " deleted configuration for default event bus.");
```

Comment:
An interesting observation - one annoyance here too is that the timezone is not specified nor printed - it uses the 'default' which could be ambiguous.

That being said, it's something we can come back to another time (If so, suggest closing this feedback). May be worth considering an issue to check how we use dates

Reply:
agreed

Reply:
The timezone is not included in the message so that machines running in different timezones produce consisent messages - we can not assume the administrator is local to the server.